### PR TITLE
Add documentation for the new API.

### PR DESCRIPTION
**Pull request by mplooster**

### CODE REVIEW

File: docs-src/v2/content/sessions/update.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+{{< tab "Java/Android" "java" >}}
+// Example coming soon!
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Process Command Result
+After receiving a [Command Action](../actions/#command-action), the SDK
+app should always update the session with a command result.
```
Comment (renaissanceGeek) [inline docs-src/v2/content/sessions/update.md:78]:
```suggestion
After receiving a [Command Action](../actions/#command-action), the client
app performs the appropriate business logic, then should always update the session with the result.
```

File: docs-src/v2/content/sessions/update.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+{{< tab "Java/Android" "java" >}}
+// Example coming soon!
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Set Story
+The `SetStory` method of the API may be used to update a session to run
+from a specified "story" in the model. This method is particularly useful
```
Comment (renaissanceGeek) [inline docs-src/v2/content/sessions/update.md:128]:
```suggestion
The `SetStory` method of the API may be used to update a session to begin running
from a specified "story" in the model. This method is particularly useful
```

**Review by renaissanceGeek — APPROVED** [review]
I'm not sure about this, but I *think* we could just move `docs-src/content` to `docs-src/content/v1` and not have to duplicate everything under `static`.  

Other than that, I have some suggestions inline, but overall, it's looking great!

**Review by jhollowayj — COMMENTED** [review]
I would suggest changing `SDK App`, `client app`, `client`, `app` => `client application`.
Would it be helpful to define the overall archetecture of the deployment when we introduce diatheke?  Something like `Diatheke` <-> `Server w/ extra business logic` <-> `End User application`, but mentioning that parts 2&3 can be lumped into a single program?  Then say that the Server/EndUserApp would be referred to as "Client Application" in the rest of the documentation.

I would like to see more `Note:` and `Tip:` blocks.  The content of those is already there most of the time.  I think that wrapping them in an extra block helps them stand out a little bit more as suggestions and notes, instead of just the underlying explanation.

Otherwise, this seemed pretty good so far.

File: docs-src/v2/content/sessions/update.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mplooster) [reply docs-src/v2/content/sessions/update.md:91]: Yes, each CommandAction maps to a single `client.ProcessCommandResult` call. The updates in your example are essentially system initiated alerts, so the way to handle them would be to have the application use the `SetStory` method.

**Review by jhollowayj — APPROVED** [review]

**Pull request merged**